NotebookExamples/fsharp/Samples/Repo Statistics.ipynb

```diff
@@ -82,79 +116,105 @@
     "        State = ItemStateFilter.All)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Start pulling data via the GitHub API"
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": 62,
+   "execution_count": 54,
    "metadata": {},
    "outputs": [],
    "source": [
-    "let createdThisMonth = await (fun () -> gitHubClient.Issue.GetAllForRepository(organization, repositoryName, createdIssuesRequest) )\n",
-    "let closedThisMonth =  await (fun () -> gitHubClient.Issue.GetAllForRepository(organization, repositoryName, closedIssuesRequest) )\n",
-    "let thisYearIssues =   await (fun () -> gitHubClient.Issue.GetAllForRepository(organization, repositoryName, thisYearIssuesRequest) )"
+    "let (createdThisMonth, closedThisMonth, thisYearIssues) =\n",
+    "    async {\n",
```

Comment:
Basically this just unwinds the function await.  So, it's not clear to me this an improvement.

Reply:
The `await` function is not idiomatic.

Reply:
Lol ...

Reply:
It's not. Awaiting a task only to wrap it again so that you can block the calling thread with `.Result` is not idiomatic F# code.

Reply:
`Async.RunSynchronously` is how you block on the result.